src/components/ComboButton/_index.scss

```diff
@@ -4,9 +4,109 @@
 /// @copyright IBM Security 2019
 ////
 
+@import '@carbon/layout/scss/mini-unit';
+
+@import '@carbon/layout/scss/spacing';
+@import '@carbon/themes/scss/tokens';
+
+@import 'carbon-components/scss/globals/scss/helper-mixins';
+
 @import '../Component/mixins';
-@import 'mixins';
 
 @include component($name: combo-button) {
-  @include combo-button;
+  $root: &;
```

Comment:
We're putting all the styles in the `_index.scss` file here - definitely cleaner but can you explain the reasoning. Thanks!

Reply:
This was a continuation of the conversation on improving comprehension and reducing complexity. I refactored to this approach in areas where it was minimal work to do so, with the hope of reducing the burden later down the line (i.e. not requiring a larger long-term effort like #44).